src/apps/lwaftr/fragmentv4.lua

```diff
@@ -0,0 +1,100 @@
+module(..., package.seeall)
+
+local constants = require("apps.lwaftr.constants")
+local packet = require("core.packet")
+local ffi = require("ffi")
+
+local band, bor, lshift = bit.band, bit.bor, bit.lshift
```

Comment:
Although the BitOp module is always visible, LuaJIT docs recommend to make its inclusion explicit http://bitop.luajit.org/api.html

```lua
local bit = require("bit")
```